cloudml-template/template/trainer/model.py

```diff
@@ -26,6 +26,35 @@
 # ****************************************************************************************
 
 
+def metric_fn(labels, predictions):
+    """ Defines extra evaluation metrics to canned and custom estimators.
+    By default, this returns an empty dictionary
+
+    Args:
+        labels: A Tensor of the same shape as predictions
+        predictions: A Tensor of arbitrary shape
+    Returns:
+        dictionary of string:metric
+    """
+    metrics = {}
+
+    # Example of implementing Root Mean Squared Error for regression
+
+    # pred_values = predictions['predictions']
+    # metrics['rmse'] = tf.metrics.root_mean_squared_error(labels=labels,
+    #                                                      predictions=pred_values)
+
+    # Example of implementing Sensitivity at Specificity for classification
```

Comment:
The commented out code below seems to be for `mean_per_class_accuracy`.

Reply:
OK, the commented out code is changed to:

 # Example of implementing Mean per Class Accuracy for classification